firmware/user/modes/mode_flight.c

```diff
@@ -305,6 +336,25 @@ static void ICACHE_FLASH_ATTR flightMenuCb(const char* menuItem)
     {
         flightStartGame(FL_ENV);
     }
+    else if ( fl_flight_invertY0_env == menuItem )
+    {
+        flightSimSaveData_t * sd = getFlightSaveData();
+        sd->flightInvertY = 1;
+        setFlightSaveData( sd );        
+        flightRefreshMenu();
+    }
+    else if ( fl_flight_invertY1_env == menuItem )
+    {
+        flightSimSaveData_t * sd = getFlightSaveData();
+        sd->flightInvertY = 0;
+        setFlightSaveData( sd );
+        flightRefreshMenu();
```

Comment:
This resets the menu position, which feels weird to me. I bet this would work better if we swapped the `char*` for the label instead. I'll have to look into how to do that.